dotnet/samples/Concepts/Plugins/CopilotAgentBasedPlugins.cs

```diff
@@ -102,31 +102,55 @@ private void WriteSampleHeadingToConsole(string pluginToTest, string functionToT
         [
             "@odata.type",
             "attachments",
+            "allowNewTimeProposals",
             "bccRecipients",
             "bodyPreview",
+            "calendar",
             "categories",
             "ccRecipients",
+            "changeKey",
             "conversationId",
+            "coordinates",
             "conversationIndex",
+            "createdDateTime",
+            "discriminator",
+            "lastModifiedDateTime",
+            "locations",
             "extensions",
             "flag",
             "from",
             "hasAttachments",
+            "iCalUId",
             "id",
             "inferenceClassification",
             "internetMessageHeaders",
+            "instances",
+            "isCancelled",
             "isDeliveryReceiptRequested",
             "isDraft",
+            "isOrganizer",
             "isRead",
             "isReadReceiptRequested",
             "multiValueExtendedProperties",
+            "onlineMeeting",
+            "onlineMeetingProvider",
+            "onlineMeetingUrl",
+            "organizer",
+            "originalStart",
             "parentFolderId",
+            "range",
             "receivedDateTime",
+            "recurrence",
             "replyTo",
             "sender",
             "sentDateTime",
+            "seriesMasterId",
             "singleValueExtendedProperties",
+            "transactionId",
+            "time",
             "uniqueBody",
+            "uniqueId",
+            "uniqueIdType",
```

Comment:
Lets not forget to include the `me_calendar_CreateEvents` in line 213 of this file similar to the `DemoCommand.cs` file in this PR.